Source/Fuse.PushNotifications/Android/BigPictureStyleHttp.java

```diff
@@ -7,7 +7,7 @@
 import android.graphics.Bitmap;
 import android.graphics.BitmapFactory;
 import android.os.AsyncTask;
-import android.support.v4.app.NotificationCompat;
+import androidx.core.app.NotificationCompat;
```

Comment:
I see some new references to `androidx` in this PR. Maybe this should wait until we've merged #1323?

Reply:
oh shucks, yeah, I forgot to remove that when I was updating the previous changes, but sure, can wait for #1323